addons/SpellCheck/README.md

```diff
@@ -0,0 +1,18 @@
+#SpellCheck
+
+This addon lists spells you haven't unlocked yet.
+
+##Usage
+
+//spellcheck whm | blm | smn | nin | brd | blu | geo | tru
+
+##Credits
+shinpad, trv, Arcon et all at 
```

Comment:
Just a nit, nothing to block the pull request for, but it's actually "et al.", an abbreviation for the latin phrase "et alii" meaning "and others".